src/main/java/org/broadinstitute/consent/http/models/dto/DataSetDTO.java

```diff
@@ -128,4 +132,26 @@ public void setAlias(Integer alias) {
     public String getAlias(){
         return alias;
     }
+
+    public String getObjectId() {
+        return objectId;
+    }
+
+    public void setObjectId(String objectId) {
+        this.objectId = objectId;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Do you need this line?

Reply:
Yes, this is the standard per effective java (one example: https://www.baeldung.com/java-hashcode)